hack/istio/install-bookinfo-demo.sh

```diff
@@ -211,15 +226,34 @@ $CLIENT_EXE get services -n ${NAMESPACE}
 $CLIENT_EXE get pods -n ${NAMESPACE}
 
 # If OpenShift, we need to do some additional things
-if [[ "$CLIENT_EXE" = *"oc" ]]; then
+if [[ "${IS_OPENSHIFT}" = "true" ]]; then
   $CLIENT_EXE expose svc/productpage -n ${NAMESPACE}
   $CLIENT_EXE expose svc/istio-ingressgateway --port http2 -n ${ISTIO_NAMESPACE}
-  cat <<NAD | $CLIENT_EXE -n ${NAMESPACE} create -f -
+  if [[ "${IS_MAISTRA}" == "true" ]]; then
```

Comment:
You mean if NOT maistra - Maistra will create the NAD for us. If we are NOT in Maistra, then we have to do this ourselves.
```suggestion
  if [[ "${IS_MAISTRA}" != "true" ]]; then
```